osi_logicaldetectiondata.proto

```diff
@@ -0,0 +1,228 @@
+syntax = "proto2";
+
+option optimize_for = SPEED;
+
+import "osi_version.proto";
+import "osi_common.proto";
+
+package osi3;
+
+//
+// \brief Processed data from one or multiple sensors as a list of logical detections. Logical detections are derived from sensor detections in a logical model through processing steps like fusion filtering, tracking etc.
+//
+// All information is given with respect to the reference frame of the logical/virtual sensor
+// \c SensorView::mounting_position (e.g. center of rear axle of the ego car) in Cartesian coordinates.
+//
+message LogicalDetectionData
+{
+    // The interface version used by the sender (i.e. the simulation
+    // environment).
+    //
+    optional InterfaceVersion version = 1;
+
+    // Header attributes of fused detections from multiple sensors and sensor types.
+    //
+    optional LogicalDetectionDataHeader header = 2;
+
+    // logical detections consisting of transformed (and potentially fused) detections from one or multiple sensors and sensor types.
+    // Logical detections are given with respect to the reference frame of the logical/virtual sensor
+    // \c SensorView::mounting_position (e.g. center of rear axle of the ego car)
+    //
+    // \note OSI uses singular instead of plural for repeated field names.
+    //
+    repeated LogicalDetection logical_detection = 3;
+}
+
+//
+// \brief The header attributes of the list of logical detections.
+//
```

Comment:
@max-rosin is this not a better wording?
\brief Header attributes of the logical detections' list.

Reply:
I will change it to `\brief The header attributes of each sensor's logical detections list.` ok?

Reply:
Done.